binstar_client/commands/upload.py

```diff
@@ -166,6 +167,9 @@ def main(args):
 
         package_type = determine_package_type(filename, args)
 
+        if package_type == 'project':
+            sys.exit(projects_uploader(filename, args))
```

Comment:
Seeing sys.exit here is a little weird... can `main` be refactored a little bit so there is a clean branch for projects vs. packages?